src/Sentry/NdkHandlerStrategy.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sentry
+{
+    /// <summary>
+    /// Handler strategy
+    /// </summary>
+    /// 
+    public enum NdkHandlerStrategy
```

Comment:
This is Android specific, but it was added to the whole SDK regardless of platform. Please add under `/Android/`

It should only show up if you're `#if ANDROID`